### DISCUSSION TITLE
How to enable renovate in the forked repository

### DISCUSSION THREAD

**Discussion by rajadileepkolli** [renovatebot/renovate#12619] [Request Help] [answered]
I want to enable renovate in the repository which I have forked,

I have manually added the [renovate.json](https://github.com/rajadileepkolli/moviebuffs/blob/master/renovate.json), but sill I see from [logs](https://app.renovatebot.com/dashboard#github/rajadileepkolli/moviebuffs/505462628) that it is still not working and giving me error `Repository is a fork and not manually configured - skipping`

How to enable it?

**Comment by viceice** [comment, marked as answer]
Wrong property 👉  https://docs.renovatebot.com/configuration-options/#includeforks
